pbft/pbft.cpp

```diff
@@ -718,19 +705,14 @@ pbft::wrap_message(const pbft_membership_msg& msg, const std::string& /*debug_in
     return result;
 }
 
-bzn::encoded_message
-pbft::wrap_message(const audit_message& msg, const std::string& debug_info)
+bzn_envelope
+pbft::wrap_message(const audit_message& msg) const
 {
-    bzn::json_message json;
-
-    json["bzn-api"] = "audit";
-    json["audit-data"] = boost::beast::detail::base64_encode(msg.SerializeAsString());
-    if (debug_info.length() > 0)
-    {
-        json["debug-info"] = debug_info;
-    }
+    bzn_envelope result;
+    result.set_audit(msg.SerializeAsString());
+    result.set_sender(this->uuid);
```

Comment:
I added: 
     result.set_timestamp(this->now());
     this->crypto->sign(result);

to the other two wrap_message functions in KEP-747. Probably want that here too.